BPA/Ch.12/binmix.stan

```diff
@@ -0,0 +1,36 @@
+// Simplest binomial mixture model
+data {
+  int<lower=0> R;       // Number of sites
+  int<lower=0> T;       // Number of temporal replications
+  int<lower=0> y[R, T]; // Counts
+  int<lower=0> K;       // Upper bound of population size
+}
+
+transformed data {
+  int<lower=0> max_y[R];
+
+  for (i in 1:R)
+    max_y[i] <- max(y[i]);
+}
+
+parameters {
+  real<lower=0> lambda;    // Mean population size
+  real<lower=0,upper=1> p; // Detection probability
+}
+
+model {
+  // Priors
+  lambda ~ gamma(0.005, 0.005);
```

Comment:
We strongly recommend against these diffuse gamma priors as they have bad statistical and computational properties (too much mass in the tail relative to the head of the distribution, primarily).  Andrew's written several papers about it that we reference in the manual.

What do you know about the population size ahead of time?